code/aind_smartspim_stitch/path_parser.py

```diff
@@ -5,7 +5,7 @@
 from pathlib import Path
 from typing import List, Union
 
-from utils import utils
+from .utils import utils

```

Comment:
May want to be consistent with the imports in main.py and make this an absolute import.